### ISSUE TITLE
inkgd not compatible with Godot 4.2

### ISSUE THREAD

**Issue by francoisdlt** [ephread/inkgd#76]
**Describe the bug**
When running inkgd in Godot 4.2.beta1, I get the following error : 
```
res://addons/inkgd/runtime/values/value.gd:104 :
Parser Error: The function signature doesn't match the parent. Parent signature is "copy() -> Variant".
```

**To Reproduce**
Install Godot 4.2.beta.1 with the godot4 version of the plugin 

**Expected behavior**
No error :)

**Ink files**
Irrelevant

**Environment:**
 - OS: macOS 14.0 (M2 chip)
 - Godot version: Godot 4.2.beta1 vanilla
 - Inklecate version: 1.1.1
 - _inkgd_ version: 0.6.0 (godot4 branch)

**Comment by nyxkn** [comment]
I changed copy() to return Variant, and that seems to fix this. Both in ink_object.gd and in value.gd.

`func copy() -> Variant:`

**Comment by ephread** [comment]
Thanks @nyxkn and @francoisdlt, I'll fix this!

**Comment by ephread** [comment]
Fixed by 470c734.
